nodebuilder/header/service.go

```diff
@@ -8,6 +8,20 @@ import (
 	"github.com/celestiaorg/celestia-node/header/sync"
 )
 
+// API is a wrapper around Module for the RPC.
+// TODO(@distractedm1nd): These structs need to be autogenerated.
+type API struct {
+	// GetByHeight returns the ExtendedHeader at the given height, blocking
+	// until header has been processed by the store or context deadline is exceeded.
+	GetByHeight func(context.Context, uint64) (*header.ExtendedHeader, error)
+	// Head returns the ExtendedHeader of the chain head.
+	Head func(context.Context) (*header.ExtendedHeader, error)
+	// IsSyncing returns the status of sync
+	IsSyncing func() bool
```

Comment:
we don't need double docs between API struct and Module

I prefer docs on actual Module

Reply:
done